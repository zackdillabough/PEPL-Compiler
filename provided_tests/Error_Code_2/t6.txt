POLY A(R) = R + 2 R + 1;
POLY B(R) = R + 2 S + 1;
POLY M = a + 
m - x 
- x^2 + p;
POLY C(W, C, D, S, J, R, S, F, A) = 



T 
+ 
1
;

START
A(3);
INPUT X;
B(X);
INPUT Y;
INPUT Z;
INPUT X;
INPUT Y;
INPUT Z;
C(1, 2, 3, 4, 5, 6, 7, X, Y);
9 24 3 282 462 4
