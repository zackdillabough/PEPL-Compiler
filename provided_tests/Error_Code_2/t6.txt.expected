Error Code 2: 2 3 4 5 10 